src/app/models/propTypes/storyItem/index.js

```diff
@@ -7,16 +7,29 @@ export const storyItemImage = {
   width: number,
 };
 
+export const block = arrayOf({
+  type: string,
+});
+
 export const storyItem = {
   headlines: shape({
-    headline: string.isRequired,
+    promoHeadline: {
+      blocks: shape(block),
+    },
   }),
   locators: shape({
-    assetUri: string.isRequired,
+    optimoUrn: string.isRequired,
```

Comment:
Is this going to cause prop type errors for CPS? Not sure where this `storyItem` prop type is used so it might not.